ts/Stock/StockToolsBindings.ts

```diff
@@ -286,6 +290,25 @@ bindingsUtils.manageIndicators = function (
     } else {
         seriesConfig.id = uniqueKey();
         navigation.fieldsToOptions(data.fields, seriesConfig);
+        parentSeries = chart.get(seriesConfig.linkedTo);
+        defaultOptions = getOptions().plotOptions as any;
+
+        // Make sure that indicator uses the SUM approx if SUM approx is used
+        // by parent series (#13950).
+        if (
+            parentSeries &&
+            parentSeries instanceof Highcharts.Series &&
```

Comment:
For optimization consider testing with `typeof parentSeries !== 'undefined'`, as it does not go up the `super` constructor.

Reply:
Okay!

Reply:
Done 👍